Add summit recap post (#582)
trinodb/trino.io: _posts/2023-12-13-trino-summit-recap.md

---
layout: post
title: "Trino Summit 2023 recap"
author: "Manfred Moser, Cole Bowden"
excerpt_separator: <!--more-->
image: /assets/blog/trino-summit-2023/summit-logo.png
show_pagenav: false
---

> Two days of non-stop Trino action completed! Last week Trino Summit 2023

Reviewer: ```suggestion
Two days of non-stop Trino action are done! Last week, Trino Summit 2023
```

> happened as another great community event. Great presentations from Trino

Reviewer: "happened" feels a little passive

Author: took place virtually ? jhahaha

went off ?

experts across the globe showed different use cases and experiences with Trino.

<!--more-->

> During the event the lively audience of over 600 attendees asked questions from

Reviewer: ```suggestion
During the event, our lively audience of over 600 attendees asked questions from
```

> the speakers and each other on chat, and had fun with Trino trivia questions.

Reviewer: ```suggestion
the speakers and each other on chat, and we had fun with Trino trivia questions.
```

We talked about the [SQL routine competition]({% post_url 2023-11-09-routines
%}) and announced Kevin Liu from Stripe and Jan Was from Starburst as the
winners. You can find their submissions in [the examples page for SQL
routines]({{site.url}}/docs/current/routine/examples.html).

Starburst announced their [Trino Champions
program](https://www.starburst.io/community/trino-champions/). Kevin and Jan are
the first recipients of the award and will receive their swag packs soon. Going

> forward new champions will be crowned regularly, and Starburst is [looking for

Reviewer: ```suggestion
forward, new champions will be crowned regularly, and Starburst is [looking for
```

nominations](https://www.starburst.io/community/trino-champions/).

## Sessions

If you missed out on the event, the following list of all the sessions provides

> links to the recordings. Over time we will follow up with blog posts about each

Reviewer: ```suggestion
links to the recordings. Over time, we will follow up with blog posts about each
```

  presented by Gabriel Fernandes de Oliveira and Yifan Pan from
  [Quora](https://www.quora.com/).
* [Inherent race condition in Guava Cache invalidation and how to escape it]()
  presented by Piotr Findeisen from
  [Starburst](https://www.starburst.io/).
* [Unstructured data analysis using polymorphic table function in Trino]()
  presented by YongHwan Lee from
  [SK Telecom](https://www.sktelecom.com).
* [Transitioning to Trino: Evaluating Lyft’s query engine capabilities]()
  presented by Charles Song from
  [Lyft](https://www.lyft.com/).
* [Visualizing Trino with Apache Superset](https://www.youtube.com/watch?v=idk0GMxs8vE)
  presented by Evan Rusackas from
  [Preset](https://preset.io/).
* [Trino OPA authorizer - An open source love story]()
  presented by Sönke Liebau ([Stackable](https://stackable.tech/))
  and Pablo Arteaga ([Bloomberg](https://www.bloomberg.com)).
* [VAST database catalog]()
  presented by Jason Russler from
  [VAST](https://vastdata.com/).
* [Support for Parquet decryption and aggregate gushdown In Trino]()
  presented by Amogh Margoor and Manish Malhotra from
  [Apple](https://www.apple.com/).

> Shout out for all their work with the speakers and organizing the event go to

Reviewer: ```suggestion
Shout outs for all their work with the speakers and organizing the event go to
```

Anna Schibli, Mandy Darnell, and Monica Miller from the Trino Summit event team,
and everyone else at Starburst who helped make this event a success.

> Special thanks for making Trino Software Foundation event a

Reviewer: ```suggestion
Special thanks for making this Trino Software Foundation event a
```
